js/models/conversations.js

```diff
@@ -675,7 +675,8 @@
         return collection.fetchConversation(this.id, 1).then(function() {
             var lastMessage = collection.at(0);
             if (lastMessage) {
-                if (lastMessage.get('type') === 'verified-change') {
+                var type = lastMessage.get('type');
+                if (type === 'verified-change' || lastMessage.get('expirationTimerUpdate')) {
```

Comment:
Minor: How about creating a explanatory constant such as `const shouldSkipTimestampUpdate = type === 'verified-change' || lastMessage.get('expirationTimerUpdate')`?

Also: Would you be open to use `const` and `let` instead of `var` for new code? It might help prevent subtle bugs when `var`s change that shouldn’t 😄

Reply:
For a small bugfix, no I don't think I want to move this method over to the new syntax. And I don't want to start using `const`/`let` until we move over to the new syntax. When we do feature work in this space, we'll move it.

Will introduce the explaining variable.